src/test/java/tests/FormEntryTest.java

```diff
@@ -125,5 +127,18 @@ public void testFormEntry() throws Exception {
         response = answerQuestionGetResult("11", "1 2 3", sessionId);
 
         System.out.println("result3: " + response);
+
+        ObjectMapper mapper = new ObjectMapper();
+
+        CurrentRequestBean currentRequestBean = mapper.readValue
+                (FileUtils.getFile(this.getClass(), "requests/current/current_request.json"), CurrentRequestBean.class);
+        currentRequestBean.setSessionId(sessionId);
+
+        //Test Current Session
+        ResultActions currentResult = mockMvc.perform(get("/current")
```

Comment:
not sure how spring does URL routing, but wondering if there's a way to get URLs that doesn't involve typing out the hard coded string